src/main/java/uk/gov/hmcts/reform/sscs/transformers/SscsCaseTransformer.java

```diff
@@ -136,14 +150,19 @@ public CaseResponse transformExceptionRecord(ExceptionRecord exceptionRecord, bo
 
         transformed.put("caseCreated", scannedData.getOpeningDate());
 
+        if (FormType.SSCS2.toString().equalsIgnoreCase(formType)) {
```

Comment:
As per comment above, think this can be moved to `SscsDataHelper`

Reply:
Moved